src/Environments.jl

```diff
@@ -0,0 +1,24 @@
+# Contains the structs that define the static and dynamic environment that configures/forces the river model.
+export StaticEnvironment, DynamicEnvironment, Environment
+
+struct StaticEnvironment
+    # static data objects
+    routing_lv_basins_dir::String
```

Comment:
Im confused why we call this routing_lv_basins_dir, because it points to a list of level 5 basins (hydrosheds level, not routing level). 

Here we seem to assign a routing level based on line number in the file? 
```
# Auxiliary function
## function for reading basins from txt file into Vector{Int}
function read_routing_lv_basins(routing_lv_file::String)
    routing_lv_basins = Int64[]
    file = open(joinpath(routing_lv_file))
    for line in eachline(file)
        push!(routing_lv_basins, parse(Int64, line))
    end
    close(file)

    return routing_lv_basins
end
```

but I dont think this is correct given the graph

Reply:
Based on the graph, is this correct?
1051429650 1
1051429580 1
1051435110  1
1051435100 2
1051460430 3

Reply:
We can address this in a future PR (naming)